redhat/frr.spec.in

```diff
@@ -254,14 +254,14 @@ developing OSPF-API and frr applications.
     --disable-ospfclient\
 %endif
 %if %{with_ospfapi}
-    --enable-ospfapi=yes \
```

Comment:
I think this commit is wrong.

=yes means enable
=no means disable and the removal on both sides of the if/else here seems like it would not work.

Reply:
You're absolutely right. I did have a commit which changed the else case to --disable but somehow dropped it from my PR. Will add a followup commit to fix this.